x/sequencer/keeper/msg_server_unbond.go

```diff
@@ -0,0 +1,50 @@
+package keeper
+
+import (
+	"context"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/dymensionxyz/dymension/v3/x/sequencer/types"
+)
+
+// Unbond defines a method for removing coins from sequencer's bond
+func (k msgServer) Unbond(goCtx context.Context, msg *types.MsgUnbond) (*types.MsgUnbondResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	completionTime, err := k.setSequencerToUnbonding(ctx, msg.Creator)
```

Comment:
if there are 2 sequencers, one bonded and one proposer, and the proposer is unbonding the other should automatically become the proposer.

Reply:
right
handled in separate PR